backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/CassandraCluster.java

```diff
@@ -42,10 +42,12 @@
 
     private static final long SLEEP_BEFORE_RETRY = 200;
     private static final int MAX_RETRY = 200;
+    private static final long EMBEDDED_CASSANDRA_TIMEOUT = MAX_RETRY * SLEEP_BEFORE_RETRY;
```

Comment:
I would prefer an independent value, as the MAX_RETRY & SLEEP_BEFORE_RETRY don't make sense for this, IMHO.
You may also add the unit in the constant name.